### PR TITLE
Update amplitude cutoff documentation

### PR DESCRIPTION
**Pull request by jsiegle**
Edit documentation for amplitude cutoff metric to better reflect the SpikeInterface implementation.

### CODE REVIEW

File: doc/modules/qualitymetrics/amplitude_cutoff.rst [hunk 1]
```diff
@@ -6,15 +6,16 @@ Amplitude cutoff (:code:`amplitude_cutoff`)
...
+
+The calculation works best when the amplitude distribution is *symmetric* and has a *single peak*. Importantly, these assumptions likely do not hold if there is substantial drift. In this case, amplitude cutoff can be computed separately for different chunks of the recording, yielding a more accurate estimate of the fraction of spikes missing from each chunk.
+
+**IMPORTANT:** When applying this metric, it is critical to know how the spike amplitudes were calculated. Template-based sorting algorithms such as Kilosort output the template scaling factors that are used to fit particular templates to each spike waveform. By multiplying the template amplitude by the scaling factor, one can approximate the original amplitude of each spike. However, this amplitude often does not match the true amplitude. Amplitude cutoffs computed from the template scaling factors (`amplitudes.npy` in the Kilosort output) tend to be much higher than when using actual spike amplitudes extracted from the raw data. It's important to know which method was used in order to compare amplitude cutoff values across experiments.
```
Comment (zm711) [inline doc/modules/qualitymetrics/amplitude_cutoff.rst:18]: I guess my only question here is that spikeinterface isn't using `amplitudes.npy` so do we want to mention that here?

**Review by zm711 — COMMENTED** [review]
See question.

File: doc/modules/qualitymetrics/amplitude_cutoff.rst (hunk 1 above)
Comment (jsiegle) [reply doc/modules/qualitymetrics/amplitude_cutoff.rst]: Yep, just added a sentence about that!

Comment (zm711) [reply doc/modules/qualitymetrics/amplitude_cutoff.rst]: Thanks. That looks good to me!

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**